bin/deploy

```diff
@@ -171,7 +174,7 @@ cat <<EOF
 
 Done. Run the following:
 
-    . $LSSTSW/bin/setup.sh
+    . ${LSSTSW}/bin/setup.sh
```

Comment:
I know. It's what you want people to copy and paste into their shell. So technically they would need the quotes wouldn't they?